lua/barbar/config.lua

```diff
@@ -119,11 +121,13 @@ local ICON_PRESETS = {
   powerline = function(default_icons)
     default_icons.inactive = { separator = { left = '', right = '' } }
     default_icons.separator = { left = '', right = '' }
+    default_icons.separator_at_end = false
```

Comment:
Excellent usage of `preset`s, I was doing to just remove it entirely but this is much more elegant.

Reply:
Yeah, I first added the option with a static default value and then noticed it works much better in combinatin with presets.